api-graphql/graph/model/exam.go

```diff
@@ -0,0 +1,53 @@
+package model
+
+type Exam interface {
+	IsExam()
+}
+
+type Outcome interface {
```

Comment:
Similar feedback here, I would use methods here that reflect the details or actions of an Outcome beyond simple identification.